.github/workflows/ci.yml

```diff
@@ -376,4 +394,25 @@ jobs:
         run: composer bin psalm install
 
       - name: Run static analysis
-        run: bin/tools/psalm/vendor/vimeo/psalm/psalm --output-format=github --php-version=7.4
+        run: bin/tools/psalm/vendor/vimeo/psalm/psalm --output-format=github --php-version=7.2
+
+  docker-stack:
+    name: CI with docker stack
+    continue-on-error: true
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          # - windows-latest
```

Comment:
:man_shrugging: 

I have zero skills in windows, so I cannot make this one work